src/Atmos/Model/energy.jl

```diff
@@ -0,0 +1,97 @@
+export AbstractEnergyModel, EnergyModel, θModel
+#### Energy component in atmosphere model
+abstract type AbstractEnergyModel <: BalanceLaw end
```

Comment:
Does this have to be a `BalanceLaw`?

Reply:
No, I just added so that we don't need to define `vars_state` for all `AbstractStateType`'s

Reply:
I removed the BL subtype